routes/campaigns/campaign.php

```diff
@@ -173,3 +173,15 @@
 
 Route::get('/w/{campaign}/confirm-delete', [App\Http\Controllers\ConfirmController::class, 'index'])->name('confirm-delete');
 Route::post('/w/{campaign}/vanity-validate', [App\Http\Controllers\Campaign\VanityController::class, 'index'])->name('campaign.vanity-validate');
+
+// Permission save
+Route::get('/w/{campaign}/webhooks', [App\Http\Controllers\Campaign\WebhookController::class, 'index'])->name('webhooks.index');
+Route::get('/w/{campaign}/webhooks/create', [App\Http\Controllers\Campaign\WebhookController::class, 'create'])->name('webhooks.create');
```

Comment:
Delete most of these by using the `Route::resources([` code at the beginning of this file